client/jetpack-cloud/sections/partner-portal/credit-card-fields/index.tsx

```diff
@@ -69,8 +69,8 @@ export default function CreditCardFields() {
 		},
 	};
 
-	const { data: { items: paymentMethods } = [], isFetching: isFetchingPaymentMethods } =
-		useRecentPaymentMethodsQuery();
+	const { data = [], isFetching: isFetchingPaymentMethods } = useRecentPaymentMethodsQuery();
+	const { items: paymentMethods } = data as any;
```

Comment:
Unfortunately needed to add a runtime change 😭 
There's no way to squeeze the `any` cast into the destructuring syntax.